weave

```diff
@@ -1197,6 +1202,9 @@ launch() {
                 IPRANGE="${1#*=}"
                 IPRANGE_SPECIFIED=1
                 ;;
+            --hairpin-mode=*)
+                HAIRPIN_MODE="${1#*=}"
```

Comment:
I could understand this argument if `--hairpin-mode` did anything in non-CNI modes, or if you installed via `weave setup`, or maybe some other cases.  

But the only thing it does is affects the CNI config file generated the first time you run `weave launch`, which I find too specific to be worth doing.